kratos/includes/condition.h

```diff
@@ -315,20 +315,26 @@ class Condition : public GeometricalObject
      */
     virtual void GetValuesVector(Vector& values, int Step = 0)
     {
+        if (values.size() != 0)
```

Comment:
please add the parenthesis:
~~~c++
if (values.size() != 0) {
    values.resize(0, false);
}
~~~